pkg/webhook/server.go

```diff
@@ -57,6 +59,10 @@ type Server struct {
 	// CertName is the server key name. Defaults to tls.key.
 	KeyName string
 
+	// CAName is the CA certificate name which server used to verify remote(client)'s certificate.
+	// Defaults to "", which means server does not verify client's certificate.
+	CAName string
```

Comment:
Name is a little confusing.
Probably `ClientCAName` is better.

Reply:
done, thanks!